packages/lib/src/vis/heatmap/models.ts

```diff
@@ -29,3 +30,8 @@ export interface ScaleShader {
   ) => Record<string, { value: number }>;
   fragment: string;
 }
+
+export type CompatibleTypedArray = Exclude<
+  TypedArray,
+  Uint8ClampedArray | Float64Array
```

Comment:
I don't know if they are always present depending on JS version, but `BigInt64Array` and `BigUInt64Array` should also be excluded.

Reply:
`TypedArray` comes from `@types/ndarray`, in which bigint types are treated separately for backwards compatibility (because only the latest versions of TS know what bigints are).